insights/client/connection.py

```diff
@@ -439,9 +439,13 @@ def handle_fail_rcs(self, req):
                         req.status_code)
             logger.debug("HTTP Status Text: %s", req.reason)
             if req.status_code == 401:
-                logger.error("Please ensure that the system is registered "
-                             "with RHSM for CERT auth, or that correct "
-                             "credentials are set in %s for BASIC auth.", self.config.conf)
+                logger.error("This machine has not yet been registered, "
+                             "\nplease ensure that the system is registered with subscription-manager "
+                             "and then with insights-client."
+                             "\n\n. Register with subscription-manager"
+                             "\n# subscription-manager register"
+                             "\n\n. Register with insights-client"
+                             "\n# insights-client --register")
```

Comment:
1. I'd remove the newline before `please ensure`..., let's keep it on one line.
2. The output contains dots before `Register with`..., is that intentional? It feels strange.

```
. Register with subscription-manager
# subscription-manager register

. Register with insights-client
# insights-client --register
```

Reply:
I was typing up my review but you beat me to it @m-horky !

I have the same thoughts as Matyas here, and I believe there are supposed to be numbers preceding the dots:
```
1. Register with subscription-manager
# subscription-manager register

2. Register with insights-client
# insights-client --register
```

Reply:
Hi,
Number typos are corrected.
Displaying of new/old message about cert/basic auth method is now treated by else-if statement checking value of `config.authmethod`.

Reply:
Thanks, I'll review this soon :+1: